modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/SampleData_vertebrates_conf.pm

```diff
@@ -0,0 +1,68 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=head1 NAME
+
+Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_vertebrates_conf
+
+=head1 SYNOPSIS
+
+
+=head1 DESCRIPTION
+
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_vertebrates_conf;
+
+use strict;
+use warnings;
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_conf');
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Hive::Version 2.5;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+
+
+=head2 default_options
+
+ Description: It returns a hashref containing the default options for HiveGeneric_conf
+ Returntype : Hashref
+ Exceptions : None
+
+
+=cut
```

Comment:
I realise that this has just been copied from the nonvert config, but none of this is required - it doesn't hurt, but there's no point in pulling in libraries that you don't need, or having documentation that doesn't really tell you anything useful...

Reply:
Agree! Fixed in 5784cf8